src/main/java/pageobjectsnew/CartPageNew.java

```diff
@@ -0,0 +1,62 @@
+package pageobjectsnew;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static utility.Services.WaiterService.waitForTextVisible;
+import static utility.Services.WebElementService.clickOnElement;
+
+/**
+ * Created by user on 31.05.17.
+ */
+public class CartPageNew {
+
+    protected WebDriver driver;
+
+    public CartPageNew(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(id = "page-title")
+    public WebElement cartItemsNumber;
+
+    @FindBy(css = ".item-title a")
+    public List<WebElement> itemsNames;
+
+    @FindBy(xpath = "//*[@class='remove next-previous-assigned']")
```

Comment:
No, className will not work - invalid selector: Compound class names not permitted
Refactor it to easier css locator